packages/devtools_app/test/inspector_layout_tab_test.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:devtools_app/src/inspector/flutter/layout_tab.dart';
+import 'package:devtools_app/src/ui/fake_flutter/_real_flutter.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('FlexProperties.fromJson creates correct value from enum', () {
+    for (var direction in Axis.values) {
```

Comment:
wow this test is checking a lot of cases. it would also be fine to check one property at a time as the fields don't interact but as long as the test runs quickly it is fine as is.

Reply:
because of your suggestion to create `EnumDeserializer` this is no longer required, thanks Jacob!